float32 radar1
float32 radar2
float32 radar3
float32 radar4
float32 radar5
float32 radar6
float32 radar7
float32 radar8
